app/[addressOrDomain]/page.tsx

```diff
@@ -555,8 +555,14 @@ export default function Page({ params }: AddressOrDomainProps) {
         fetchPortfolioAssets(data),
       ]);
     } catch (error) {
-      showNotification("Error while fetching address portfolio", "error");
       console.log("Error while fetching address portfolio", error);
+      if (error instanceof Error && error.name === 'AbortError') {
+        // Do not show notification for AbortError
+        return;
+      }
+
+      showNotification("Error while fetching address portfolio", "error");
+
     } finally {
       setLoadingProtocols(false);
```

Comment:
I think this is why the skeleton disappear before loading everything. When the requests fail, you should not set loadingProtocols to false. You should only do that after everything has successfully loaded (so I guess at the end of the `try {` )